web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1875,10 +1876,11 @@ private BaseQueryResponse next(final String id, boolean checkForContentLookup) {
             }
             
             QueryException qe = new QueryException(DatawaveErrorCode.QUERY_NEXT_ERROR, e, MessageFormat.format("query id: {0}", id));
-            log.error(qe);
+            log.error(qe, e);
```

Comment:
I'm not sure we want to log the full stack trace to the error channel if this is a wrapped NoResultsException. (We may also not want to add the exception to the response in that case either.)

Reply:
agreed